HW_4/task_7.py

```diff
@@ -0,0 +1,21 @@
+''' Задание №7
+Функция получает на вход словарь с названием компании в качестве ключа 
+и списком с доходами и расходами (3-10 чисел) в качестве значения.
+Вычислите итоговую прибыль или убыток каждой компании. 
+Если все компании прибыльные, верните истину, а если хотя бы одна убыточная — ложь.
+'''
+
+
+def calculate_profit(dictionary):
+    return all(sum(value for value in values if value > 0) >= abs(sum(value for value in values if value < 0))
+               for values in dictionary.values())
```

Comment:
Что-то вы очень усложнили
```suggestion
    return all(map(lambda lst: sum(lst) > 0, dictionary.values()))
```